packages/core/src/server/service.ts

```diff
@@ -142,12 +141,14 @@ export class ServerService {
   }
 
   reload({ graphqlSchema }: { graphqlSchema: GraphQLSchema }) {
+    const SERVER_URL = "http://localhost:42069";
```

Comment:
This approach might be tricky - won't this break in non-local environments, or if the port is different?

Reply:
it for sure will. What to use instead?

Reply:
now passing dynamic server url